raiden/transfer/mediated_transfer/target.py

```diff
@@ -217,7 +219,7 @@ def handle_onchain_secretreveal(
             secret_reveal_block_number=state_change.block_number,
         )
 
-        target_state.state = 'reveal_secret'
+        target_state.state = 'onchain_unlock'
```

Comment:
This should go to:
https://github.com/raiden-network/raiden/blob/6782cc536c29f9bebbb9574fde43e425515c2795/raiden/transfer/mediated_transfer/state.py#L286-L290